src/regions/ca/info/ca-on/clinics.fr.md

```diff
@@ -4,4 +4,6 @@ L'Ontario va ouvrir six centres d'évaluation COVID-19 dans les prochains jours,
 
 La première vague de centres d'évaluation sera située sur le site de Peel Memorial du système de santé William Osler, à l'hôpital d'Ottawa, à l'hôpital général de North York, au Mackenzie Health à Richmond Hill et au sein du réseau de santé Scarborough et du réseau Trillium Health Partners à Mississauga.
 
+Selon cette [source](https://www.blogto.com/city/2020/03/map-coronavirus-test-centres-toronto/), l'Ontario va lancer un outil d'auto-évaluation en ligne COVID-19 dans les prochains jours.
```

Comment:
This is it: https://www.ontario.ca/page/2019-novel-coronavirus-covid-19-self-assessment

Reply:
Fixed!